question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -828,7 +828,7 @@ def string_to_2darray(s, allow_complex=True):
                     A[i, j] = ans
                 except Exception:
                     # Return error if entry could not be converted to float or complex
-                    return (None, {'format_error': f'Entry ({i + 1}, {j + 1}) of matrix "{s_row[j]}" has invalid format.'})
+                    return (None, {'format_error': f'Entry "{s_row[j]}" at ({i + 1}, {j + 1}) of matrix has invalid format.'})
```

Comment:
Maybe it's worth being even more explicit here? Something like the following?
```suggestion
                    return (None, {'format_error': f'Entry "{s_row[j]}" at location (row={i + 1}, column={j + 1}) in the matrix has invalid format.'})
```